### ISSUE TITLE
Side nav anchor links not working

### ISSUE THREAD

**Issue by vardhi** [clenemt/docdash#113]
After generating docs using docdash-2.0.0, method names that appear in the side nav links to incorrect anchor.

currently generated anchors: /docs/foo_class.html#.bar_method#bar_method
expected correct link: /docs/foo_class.html#bar_method

After debugging a bit found out that, this commit (https://github.com/clenemt/docdash/commit/6306090ddd61e81ddab1e89781d97b6f72de6ab6) was causing the issue.

**Comment by hugovidafe** [comment]
Yeah, I've the same problem

**Comment by ookjosh** [comment]
Same issue using 2.0.0.

Commenting/removing the following lines from publish.js (per @vardhi link to 6306090) fixed the issue for me, at least in my minimal testing. Not sure if there are edge cases to be considered, but it appears `linkto` is working correctly (per the authors comment)?

```js
// publish.js ~ line 388
// Remove the following block:

// Not certain if there is a situation where this is needed anymore
// linkto method seems to solve this problem already, but at least this check
// will prevent duplicate #'s in links.
if (!navItemLink.includes("#." + method.name)) {
   var strNewLink = '.html#.' + method.name;

   navItemLink = navItemLink.replace(".html", strNewLink);
}

```

**Comment by demmings** [comment]
Hey @ookjosh,

I setup docdash and was about to throw it away since the links were hit and miss.  I tried your change and it is much improved.  Will keep using.  Thanks.

**Comment by ar2rsawseen** [comment]
Hello,
Sorry, I was on Christmas vacation for a couple of last weeks.
Fixed and published 2.0.1
Thank you for reporting
